src/main/java/codeu/model/data/Hashtag.java

```diff
@@ -0,0 +1,89 @@
+// Copyright 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package codeu.model.data;
+
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+/** Class representing a HSashtag. */
+public class Hashtag {
+
+  private final UUID id;
+  private final String content;
+  private List<String> userSource;
+  private List<String> conversationSource;
+  private final Instant creation;
+
+  /** Constructs a new Hashtag. */
+  public Hashtag(String content) {
```

Comment:
This implementation does not work properly with Loading of the datastore. Meaning, every time you load a hashtag, you change its id, creationTime, userSource(to A brand new list), conversationSource(to A brand new list). You should instead follow how message, user, etc.,were created. Example:
`public Conversation(UUID id, UUID owner, String title, Instant creation) {
    this.id = id;
    this.owner = owner;
    this.creation = creation;
    this.title = title;
  }`

Reply:
Ok, I overlooked that. Fixed!